Base/BacnetDailySchedule.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO.BACnet.Serialize;
+using System.Linq;
+using System.Text;
+
+namespace System.IO.BACnet
+{
+    public struct BacnetDailySchedule : ASN1.IEncode, ASN1.IDecode
+    {
+        public List<BacnetTimeValue> DaySchedule;
+
+        
+
+        public int Decode(byte[] buffer, int offset, uint count)
```

Comment:
Code cosmetics. Please remove extra new lines you added in some places like this, max one empty line is ok. And when you see a line wrapping here, then it means it's a bit too long, please try to make it shorter.